.github/workflows/snakemake-ci.yml

```
@@ -28,25 +28,29 @@ jobs:
         with: 
           environment-file: recast_helloworld/environment.yml
 
-      - name: List installed environment dependencies 
-        run: | 
-          micromamba list
-
-      - name: Linting
-        uses: snakemake/snakemake-github-action@v1
-        with:
-          directory: '.test'
-          snakefile: 'workflow/Snakefile'
-
       - name: Testing
         uses: snakemake/snakemake-github-action@v1
         with:
-          directory: '.test'
-          snakefile: 'workflow/Snakefile'
+          directory: 'recast_helloworld'
+          snakefile: 'recast_helloworld/Snakefile'
           args: '--cores 1 --use-conda --conda-cleanup-pkgs cache'
 
       - name: Create container file
         uses: snakemake/snakemake-github-action@v1
         with:
-          snakefile: 'workflow/Snakefile'
+          snakefile: 'recast_helloworld/Snakefile'
           task: 'containerize'
+          directory: 'recast_helloworld'
+
+      - name: Uploading a snakemake logs
+        uses: actions/upload-artifact@v3
+        with: 
+          name: snakemake-log
+          path: .snakemake/log/{output}.snakemake.log
+
+      - name: Upload a Dockerfile 
+        uses: actions/upload-artifact@v3
+        with:
+          name: Dockerfile
+          path: recast_helloworld/Dockerfile
```

Do the paths

```suggestion
          path: .snakemake/log/
      - name: Upload a Dockerfile 
        uses: actions/upload-artifact@v3
        with:
          name: Dockerfile
          path: Dockerfile
```

not work? If not, then it might be worth checking to see what the full directory structure beneath the working directory is with `tree` (if `tree` isn't installed then you can install it with `apt-get` or `conda`).

---

Oh, thank you, that seems to work.